projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -668,26 +867,67 @@ export class IgxTimePickerComponent implements ControlValueAccessor, OnInit, OnD
         }
     }
 
+    /**
+     * Closes the dialog without selecting the current value.
+     * ```html
+     * <igx-dialog class="igx-time-picker__dialog-popup" [leftButtonLabel]="cancelButtonLabel" (onLeftButtonSelect)="cancelButtonClick()">
+     * //...
+     * </igx-dialog>
+     * ```
+     */
     public cancelButtonClick(): void {
         this._alert.close();
         this.selectedHour = this._prevSelectedHour;
         this.selectedMinute = this._prevSelectedMinute;
         this.selectedAmPm = this._prevSelectedAmPm;
     }
 
+    /**
+     * An accessor that returns an array of the hours currently in view.
```

Comment:
Remove 'An accessor', just 'Returns... Same thing applies for minutesInView and ampmInView methods.